jenkinsbox.yml

```diff
@@ -23,13 +20,37 @@
       drush_usage_send: 1
 
     # PHP configs.
-    cibox_ssl_enable_host: true
     php_apc_enabled_in_ini: true
 
+    # Apache config.
+    cibox_ssl_folder: /etc/apache/ssl
+    apache_create_vhosts: true
+    apache_vhosts_filename: "vhosts.conf"
+    apache_remove_default_vhost: true
+    apache_default_vhost_filename: "000-default.conf"
+    apache_vhosts:
+      - {
+        servername: "cibox",
+        serveralias: "*.*",
+        documentroot: "/var/www"
+      }
+    apache_vhosts_ssl:
+      - {
+        servername: "cibox",
```

Comment:
Are we using different role or overriding this variable when we are provisioning vagrant?